pkgs/applications/networking/cluster/kubevela/default.nix

```diff
@@ -0,0 +1,64 @@
+{ buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, lib
+, stdenv
+, testers
+, kubevela
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "kubevela";
+  version = "1.9.7";
```

Comment:
Sorry for the delay, but 1.9.8 is out.

https://github.com/kubevela/kubevela/releases/tag/v1.9.8

Reply:
Updated.